blocks/shared-styles/index.js

```diff
@@ -16,6 +16,10 @@ import SmallPromoPresentation from './_children/promos/small';
 import SmallPromoStyles from './_children/promo-helpers/small/styles';
 import ExtraLargePromoPresentation from './_children/promos/extra-large';
 import LargePromoPresentation from './_children/promos/large';
+import {
+  getNavSpecificSecondaryButtonTheme,
+  getNavSpecificPrimaryButtonTheme,
+} from './_children/button/utils/getButtonStyle';
```

Comment:
As we start to think about foundations and enhanced styling this feels like we wouldn't want this here, but more along side the block implementation as it has a very specific use case

Reply:
yeah I thought about that too. I put it here because this logic will be shared between two blocks -- identity and header nav chain block

Reply:
I'm thinking a sort of arc-utils package would be best for arc-specific logic

Reply:
That makes sense - I forgot about the header nav stuff - I was just thinking in terms of this PR with regards to that - then I think it's fine to be here for now